documentation/src/main/asciidoc/topics/con_file_based_cache_stores.adoc

```diff
@@ -4,13 +4,30 @@
 File-based cache stores provide persistent storage on the local host filesystem where {brandname} is running.
 For clustered caches, file-based cache stores are unique to each {brandname} node.
 
+For a file system that supports the Portable Operating System Interface (POSIX) standard, you can mount the file system to a pod on {openshiftshort} by using the same commands that you would use with the command line interface (CLI). 
+With this functionality, you can access storage capabilities that FCP, iSCSI, NFS, or Samba does not expose. 
+
+{brandname} defaults to using the `SoftIndexFileStore` implementation for file-based cache stores, because this implementation uses B+ trees that can improve how GC removes indexes before they consume too much memory.
```

Comment:
It does not improve GC, but the memory footprint will be less as with B+tree there is no need to keep all key's in memory

Reply:
Thanks, @wfink . Updated to:

> because this implementation uses B+ trees that reduce the number of keys that need to be stored in memory.